programs/pkey/pk_decrypt.c

```diff
@@ -78,32 +78,35 @@ int main( int argc, char *argv[] )
         polarssl_printf( "usage: pk_decrypt <key_file>\n" );
 
 #if defined(_WIN32)
-        polarssl_printf( "\n" );
+        polarssl_printf( "\n  + Press Enter to exit this program.\n" );
+        fflush( stdout ); getchar();
 #endif
 
-        goto exit;
+        return( 1 );
```

Comment:
If would would like to directly return an error code, I would suggest to use MBEDTLS_EXIT_FAILURE, which are the macros defined in platform.h to return appropriate codes to the system.